feat(trivy): Return Trivy DB update timestamp in /api/v1/metadata response

danielpacak opened the pull request:
This PR introduced the following changes:

- Bump up Trivy scanner adapter to the latest release, which includes Trivy DB update timestamp in the /api/v1/metadata response
- UI enhancement to format the `harbor.scanner-adapter/vulnerability-database-next-update-at metadata` property as date
- Added extra metadata properties to easily check adapter's config:
  - `com.github.aquasecurity.trivy.debugMode`
  - `com.github.aquasecurity.trivy.ignoreUnfixed`
  - `com.github.aquasecurity.trivy.severity`
  - `com.github.aquasecurity.trivy.skipUpdate`
  - `com.github.aquasecurity.trivy.vulnType`

Resolves: #11284 

As you can see both properties are formatted as dates:

<img width="2147" alt="harbor_trivy_db_dates" src="https://user-images.githubusercontent.com/1322923/77583895-1e67ba00-6ee2-11ea-886d-764c358bb58a.png">

heww commented:
Please check to ensure that `harbor.scanner-adapter/vulnerability-database-next-update-at` will not be returned when users skip database updates.

danielpacak commented:
> Please check to ensure that `harbor.scanner-adapter/vulnerability-database-next-update-at` will not be returned when users skip database updates.
> 
> And you can rebase with the latest master branch to fix the `CI/APITEST_DB` failing.

Good catch @heww . I'm going to fix it in harbor-scanner-trivy svc and release v0.6.1

danielpacak commented:
> > Please check to ensure that `harbor.scanner-adapter/vulnerability-database-next-update-at` will not be returned when users skip database updates.
> > And you can rebase with the latest master branch to fix the `CI/APITEST_DB` failing.
> 
> Good catch @heww . I'm going to fix it in harbor-scanner-trivy svc and release v0.6.1

@heww I fixed it in 0.7.0 and updated the PR branch.
I also added metadata properties to show the current adapter config. I think it's useful for troubleshooting.

<img width="1110" alt="fix_next_update_at" src="https://user-images.githubusercontent.com/1322923/77637705-5f011b00-6f56-11ea-9e2a-267ebcbce6d9.png">
